### PR TITLE
[IBCDPE-431] iAtlas workspace

### PR DESCRIPTION
**Pull request by BWMac**
This PR adds the iAtlas workspace to tower-prod

### CODE REVIEW

File: config/projects-prod/iatlas-project.yaml
Comment (BWMac) [inline config/projects-prod/iatlas-project.yaml:5]: Just want to double check that "SCCE" is the correct department (I went off of the HTAN workspace). And also, should I be the owner? Should someone else?

Comment (thomasyu888) [reply config/projects-prod/iatlas-project.yaml:5]: I asked these on: https://sagebionetworks.jira.com/browse/IBCDPE-431?focusedCommentId=165058.  If we don't get an answer by end of week: "SCCE" is fine, and I can be the owner.

**Review by thomasyu888 — APPROVED** [review]
✔️ I am going to pre-approve this as it looks good. Just going to wait until there is a response to your comment.

File: config/projects-prod/iatlas-project.yaml
Comment (BWMac) [reply config/projects-prod/iatlas-project.yaml]: My latest commit updates the department and owner fields per James' comment on the ticket.

**Comment by thomasyu888** [comment]
👍🏼

**Comment by BWMac** [comment]
@thomasyu888 Looks like I am not authorized to merge this myself.

**Comment by thomasyu888** [comment]
Thanks @BWMac  - I'm going to let @BrunoGrandePhD merge this one when he thinks it's ready because of what's going on with tower infra right now.  But I approve!

**Review by thomasyu888 — APPROVED** [review]

**Review by BrunoGrandePhD — APPROVED** [review]
I'll merge this once Tower is stable.

**Pull request merged**
